hoomd/md/constrain.py

```diff
@@ -44,73 +44,55 @@ def _attach(self):
         super()._attach()
 
 
-class distance(Constraint):
-    R"""Constrain pairwise particle distances.
+class Distance(Constraint):
+    """Constrain pairwise particle distances.
```

Comment:
I think an example of how this is done would be good to add since using this operation involves setting some things in the snapshot in addition to using this class. That could be done either in this docstring or in a tutorial, but I think this operation will be confusing to figure out how to use without an example.

Reply:
Explaining this is far too much to put in a docstring. The docstring states all of the needed information on where the constraint data is specified and how to apply the constraint forces. The point you bring up is general to bonds, angles, and dihedrals as well and would be good to explain in a tutorial: https://github.com/glotzerlab/hoomd-examples/issues/38